src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_afs_commands.py

```diff
@@ -25,7 +25,7 @@
 class BackupTests(ScenarioTest, unittest.TestCase):
     #@record_only()
     @ResourceGroupPreparer(location="eastus2euap", random_name_length=20)
-    @VaultPreparer()
+    @VaultPreparer(soft_delete=False)
```

Comment:
Could you please specify the context of why adding the `soft_delete=False`?

Reply:
Soft delete is enabled by default, and when running tests, it blocks vault deletion and resource group cleanup as any deleted resources are still listed as protected under it.

Reply:
It's also present in other tests: https://github.com/Azure/azure-cli/blob/ef2ecdf1c75620fcc278f2d732c5f9b86dc4a7e7/src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py#L359